<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>diff-pdf GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="file1_label">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>PDF file1 (blue)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="file1_button">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="file1_text"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="file2_label">
        <property name="text">
         <string>PDF file 2 (red)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="file2_text"/>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="file2_button">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="output_text"/>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="output_button">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="output_label">
        <property name="text">
         <string>Output file</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QCheckBox" name="compare_checkbox">
        <property name="text">
         <string>Open file after comparison</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="compar_button">
        <property name="text">
         <string>Compare</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
